CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to
 
 ### Added
 
+- Models: Add Webinar xAPI activity type
```

Comment:
It should be upper, in the `Unreleased` section

Reply:
Fixed!